File: ios/TextView.m
//
//  TextView.m
//  react-native-selectable-text
//
//  Created by JoshChen on 2023/8/18.
//

#import <Foundation/Foundation.h>
#import "TextView.h"
@implementation TextView

- (BOOL)canPerformAction:(SEL)action withSender:(id)sender {
    if (action == @selector(copy:)) return YES;
    return NO;
}
@end

File: TextView.swift
//
//  TextView.swift
//  react-native-selectable-text
//
//  Created by JoshChen on 2023/7/19.
//

import UIKit

class TextView: UITextView {
    required init?(coder aDecoder: NSCoder) {
        super.init(coder: aDecoder)
        setupTextView()
    }
    override init(frame: CGRect, textContainer: NSTextContainer?) {
        super.init(frame: frame, textContainer: textContainer)
        setupTextView()
    }
    override func canPerformAction(_ action: Selector, withSender sender: Any?) -> Bool {
        return false
    }
    func setupTextView() {
        self.isEditable = false
        self.font = UIFont.systemFont(ofSize: 18)
    }
}
